### PR TITLE
Attach `shimport` for Firefox support

### LINKED ISSUE #5: Dynamic import doesn't work in Firefox
Add a ponyfill, probably this https://github.com/lukeed/dynamic-import-ponyfill

### PR DESCRIPTION
**Pull request by lukeed**
Uses [`shimport`](https://github.com/Rich-Harris/shimport) to safely swap between native `import()` and the fallback it provides. While it may be somewhat "overkill" for the rest of the project (aka, don't need module/ESM rewrite, etc), it allows the site to be functional across all browsers for the time being.

---

Closes #5

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -33,6 +33,7 @@
     </script>
     <link rel="stylesheet" href="/index.css" />
     <link rel="stylesheet" href="/library/prism.css" />
+    <script src="https://unpkg.com/shimport"></script>
```
Comment (lukeed) [inline index.html]: Even though there's another `shimport` usage that avoids this, it doesn't want to work with `type="module"`... didn't have bandwidth to dig into that further on shimport's side of things.

For reference, I'm talking about this (from the shimport readme):

```js
function shim(url) {
  try {
    return new Function(`return import('${url}')`).call();
  } catch (tmp) {
    var s = document.createElement('script');
    s.src = 'https://unpkg.com/shimport';
    s.dataset.main = url
    document.head.appendChild(s);
  }
}
```

```js
// Throws:
TypeError: shim(...) is undefined[Learn More] index.js:21:1

    <anonymous> http://localhost:5000/index.js:21 InnerModuleEvaluation self-hosted:4362 evaluation self-hosted:4315 
```

Comment (lukejacksonn) [reply index.html]: Yeh this is exactly what I tried.. didn't think (or want to really 😝) about just adding the script tag.. but it works nicely for now eh!

**Comment by lukejacksonn** [comment]
What a hero! Thank you so much for taking the time to look into it 🙇 like you say, it is a bit overkill but the diff is not huge and so probably still a worthwhile tradeoff to keep all the firefoxes 🦊 happy until they decide to support `import` like everyone else..

**Comment by lukejacksonn** [comment]
I will merge this tonight.. going to use this branch to try test creating deploys for PRs first!

**Comment by lukeed** [comment]
Cool, happy to help~ 

So i've been looking into this further, and unfortunately it _isn't_ overkill. The only way to get a browser to process ESM imports is thru `script[type=module]` tags, which means that you can't NOT rewrite the imported contents 😕The only other alternative that I've halfway-implemented-but-aborted was to do a big juggling act of dynamically created `script[type=module]` tags and then wiring them all up, but even that requires a lot of (_far more fragile_) script-rewriting which would fall apart easily.

The actual approach I'm working on is a lot like shimport. I've had a lot of the pieces laying around for a while

**Comment by lukejacksonn** [comment]
Nice. Well I look forward to seeing what you come up with! Regards this PR; after pulling it down I discovered this 😞 https://github.com/lukejacksonn/perflink/issues/5#issuecomment-479115950

**Comment by lukeed** [comment]
Hey, so I 100% proved myself wrong 😆I'm flushing out the alternative solution(s) still as I want to be feature-complete before release. If you wanna close this, no problem~! I personally would have it _displaying_ on FF but not up to me 🙃 

Either way, I'll follow up with my library once I'm done with it (soon) – I'll be scrapping the `dynamic-import-ponyfill` repo because I can't stand that name.

**Comment by lukejacksonn** [comment]
Haha, ok! I will leave this open for now to show people we are trying to be proactive about issue #5. In the meantime I am going to have a play around with ops per second timing. That change will hopefully justify make it more worthwhile shimming dynamic import for FF – because the app would actually give results and be useful then.

**Comment by lukeed** [comment]
Releasing my solution this week. I'll follow up in a separate PR with that solution. I think you might like it :)

**Pull request closed without merging**